src/argilla/client/feedback/schemas/metadata.py

```diff
@@ -319,7 +319,7 @@ class TermsMetadataFilter(MetadataFilterSchema):
         >>> TermsMetadataFilter(name="color", values=["red", "blue", "green"])
     """
 
-    type: MetadataPropertyTypes = MetadataPropertyTypes.terms
+    type: MetadataPropertyTypes = MetadataPropertyTypes.terms.value
```

Comment:
Should we align this with the changes that you did for metadata property types?
```suggestion
    type: Literal[MetadataPropertyTypes.terms] = MetadataPropertyTypes.terms.value
```

Other question that I have is about the `Literal` definition. Since they are used for primitive values, should we define types as:
```python
    type: Literal[MetadataPropertyTypes.terms.value] = MetadataPropertyTypes.terms.value
```
Otherwise, my feeling is that we're mixing enums and string.

Reply:
Sure, we can do so! In the rest of the cases is because we need to define those that way due to the `Annotated(..., discriminator="type")` which is not needed here, but we can align yes